assets/src/api/use-get-supervisors.tsx

```diff
@@ -1,9 +1,10 @@
 import { $api } from '~/shared/api';
 import { API_ENDPOINTS } from '~/shared/const';
-import { useQuery, useQueryClient } from '@tanstack/react-query';
+import { useQuery, useQueryClient, UseQueryResult } from '@tanstack/react-query';
 import { toastManager } from '~/shared/lib/toastManager';
+import { AxiosResponse } from 'axios';
 
-export const useGetSupervisors = () =>
+export const useGetSupervisors = (): UseQueryResult<AxiosResponse<ApiSupervisor[]>> =>
```

Comment:
why is it hard typed? is it no typed automatically?

Reply:
fix 23f5a1196f2889035990d78c68d4fbfabc1c1df6